docs/platform/howto/console-fork-service.rst

```diff
@@ -24,10 +24,30 @@ Fork a service using the Aiven client (CLI)
 
 1. Prepare the command to create a new service, this will contain the new copy of your data store.
 
-2. Add the ``service_to_fork_from`` parameter to specify the service to use as the source. For example, if you want to create a fork of your ``forker`` service, and name it ``forked``, the command would be something like::
+2. Add the ``service_to_fork_from`` parameter to specify the service to use as the source. 
+Change service type accordingly with ``-t``, available options::
+        
+    cassandra
+    clickhouse
+    elasticsearch
+    flink
+    grafana
+    influxdb
+    kafka
+    kafka_connect
+    kafka_mirrormaker
+    m3db
+    m3coordinator
+    m3aggregator
+    mysql
+    opensearch
+    pg
+    redis
+
```

Comment:
would be possible to show the command to list the services? I'm concerned that this list could become outdated soon if one of the services change.

Reply:
It looks like the only way to show available options is to provide a type that doesn't exist...

> avn service create forked -t notype --plan someplan
> ERROR	command failed: UserError: Unknown service type 'notype' (available options: cassandra, clickhouse, elasticsearch, flink, grafana, influxdb, kafka, kafka_connect, kafka_mirrormaker, m3db, m3coordinator, m3aggregator, mysql, opensearch, pg, redis)

Reply:
Thank you, I think it makes sense to link to the ``avn service types`` documentation then hardcode the services. What do you think @runwuf?

Reply:
Thanks @laysauchoa and @lornajane! PR updated!